src/main/g8/default.properties

```diff
@@ -9,7 +9,7 @@ package = $organization$.$name;format="norm,word"$
 #logback_version = maven(ch.qos.logback, logback-classic, stable)
 
 scala_version = 2.12.6
-sbt_version = 1.1.4
-http4s_version = 0.18.9
+sbt_version = 1.1.5
+http4s_version = 0.18.11
 logback_version = 1.2.3
-specs2_version = 4.1.0
+specs2_version = 4.2.0
```

Comment:
This is going to clash with our arbitrary instances because it depends on the new scalacheck.  This is why http4s proper is holding back on 4.0.x.  (I can't remember what 4.1 is on.)

Reply:
Is that a problem in these apps? However we can hold ourselves back if necessary.

Reply:
It's a problem if they're using any of the scalacheck-1.13 based libraries.  This is a move that the whole ecosystem needs to make together.  scalacheck had to go first, and now specs2 (and I hope scalatest) are ready, and the ball is now in discipline and cats-laws' court.  And we're downstream of that.  I'm not using scalacheck-1.14 until this migration is complete, but asking questions upstream so we can complete it.

Then again, people can always downgrade if this applies to them, for the time it applies to them.  I'm happy either way.